.circleci/config.yml

```diff
@@ -35,15 +35,14 @@ jobs:
       - run:
           name: "Start microk8s"
           command: |
-            sudo microk8s.start
-            sudo microk8s.status --wait-ready
-            sudo microk8s.enable dns
-            kubectl get all --all-namespaces
+            (echo "microk8s.start; microk8s.status --wait-ready; microk8s.enable dns rbac ") | sudo -H -s -E  
```

Comment:
I had to use echo to pipe commands once sudo -H -s -E is executed

Reply:
nice!